.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,11 @@
+name: Integration Tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@v1.0.0
+    secrets: inherit
+    with:
+      series: jammy
```

Comment:
This is what's causing the integration tests to fail. The parameter format is an enquoted json list: `'["jammy"]'`, unfortunately strings are the only way to pass complex arguments through steps.
On the other hand, I don't think you need this. The base image to run your tests will be the one defined in the `build-on` field in `charmcraft.yaml`. This is aimed at running the integration tests once per specified `series` while addding a `--series` command line parameter (as in https://github.com/jenkinsci/jenkins-charm/blob/master/.github/workflows/ci.yaml#L41)